katas/content/multi_qubit_gates/fredkin_gate/Verification.qs

```diff
@@ -0,0 +1,26 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Katas;
+    open Microsoft.Quantum.Intrinsic;
```

Comment:
We don't need this import, as Intrinsic is part of the prelude (https://github.com/microsoft/qsharp/wiki/Differences-from-the-previous-QDK#implicit-namespace-prelude)
```suggestion
```